package.json

```diff
@@ -17,15 +17,19 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.43",
     "@reduxjs/toolkit": "^1.2.5",
+    "@testing-library/react": "^9.5.0",
     "@types/node": "^12.12.26",
     "@types/react": "^16.9.19",
     "@types/react-dom": "^16.9.5",
     "@types/react-redux": "^7.1.7",
+    "@types/react-router-dom": "^5.1.3",
     "@types/webpack-env": "^1.15.1",
     "axios": "^0.19.2",
+    "keycloak-js": "^9.0.2",
```

Comment:
Does this pull keycloak.js from https://sso.pathfinder.gov.bc.ca/auth/js/keycloak.js ? This is important, as the version of the client has to match that of the server.

Reply:
@Sybrand no but react has an official keycloak library to manage calls to keycloak server.

Reply:
If the keycloak client does not match the keycloak server version - there will be issues. This was clearly specified in the acceptance criteria, and clearly stated as best practice on the official documentation.

For more detail, please refer to the official documentation: https://www.keycloak.org/docs/latest/securing_apps/#_javascript_adapter

"A best practice is to load the JavaScript adapter directly from Keycloak Server as it will automatically be updated when you upgrade the server. If you copy the adapter to your web application instead, make sure you upgrade the adapter only after you have upgraded the server."

Reply:
@Sybrand added code to read from keycloak.js on the fly.